qa/tasks/cephfs/test_admin.py

```diff
@@ -742,6 +743,309 @@ def test_fs_rename_fails_with_mirroring_enabled(self):
+        msg = ('Error code received is not the expected error code.\n'
+               'Received error code = {cfe.exception.exitstatus}\n'
+               'Expected error code = {errno.ENOENT}')
+        self.assertEqual(cfe.exception.exitstatus, errno.ENOENT, msg)
+        self.fs.mon_manager.raw_cluster_cmd('fs', 'dump')
+
+        self.fs1.getinfo()
+        self.fs2.getinfo()
+        self.remount_fss()
+
+        # check that content of both CephFSs is unaffected by this failure.
+        self.run_rw_tests()
+        self._check_fs_name_on_fs_pools((self.fs1, self.fs2))
+        # neither FS name nor FSIDs were swapped, so both must be together
+        self.assertEqual(self._are_fsnames_and_fscids_together(), True)
+
+    def test_when_fs2_is_absent(self):
+        self.fs.mon_manager.raw_cluster_cmd('fs', 'dump')
+        with self.assertRaises(CommandFailedError) as cfe:
+            self.run_cluster_cmd(f'fs swap {self.fs.name} random_fsname_654 '
+                                  '--yes_i_really_mean_it')
+        msg = ('Error code received is not the expected error code.\n'
+               'Received error code = {cfe.exception.exitstatus}\n'
+               'Expected error code = {errno.ENOENT}')
+        self.assertEqual(cfe.exception.exitstatus, errno.ENOENT, msg)
+        self.fs.mon_manager.raw_cluster_cmd('fs', 'dump')
+
+        self.fs1.getinfo()
+        self.fs2.getinfo()
+        self.remount_fss()
+
+        # check that content of both CephFSs is unaffected by this failure.
+        self.run_rw_tests()
+        self._check_fs_name_on_fs_pools((self.fs1, self.fs2))
+        self.assertEqual(self._are_fsnames_and_fscids_together(), True)
+
+    def test_when_both_fs1_and_fs2_is_absent(self):
+        self.fs.mon_manager.raw_cluster_cmd('fs', 'dump')
+        with self.assertRaises(CommandFailedError) as cfe:
+            self.run_cluster_cmd('fs swap random_fsname_65 random_fsname_66 '
+                                 '--yes_i_really_mean_it')
+        msg = ('Error code received is not the expected error code.\n'
+               'Received error code = {cfe.exception.exitstatus}\n'
+               'Expected error code = {errno.ENOENT}')
+        self.assertEqual(cfe.exception.exitstatus, errno.ENOENT, msg)
+        self.fs.mon_manager.raw_cluster_cmd('fs', 'dump')
+
+        self.fs1.getinfo()
+        self.fs2.getinfo()
+        self.remount_fss()
+
+        # check that content of both CephFSs is unaffected by this failure.
+        self.run_rw_tests()
+        self._check_fs_name_on_fs_pools((self.fs1, self.fs2))
+        # neither FS name nor FSIDs were swapped, so both must be together
+        self.assertEqual(self._are_fsnames_and_fscids_together(), True)
+
+    # Tests for "fs swap" when mirroring is enabled on FS
+
+    def test_mirroring_enabled_for_1st_FS(self):
+        """
+        Test that "ceph fs swap" fails when either or both FSs have mirroring
```

Comment:
`that "ceph fs swap" ...` - Prefix `Test` is added by test runner.

Reply:
I don't understand this comment.

Reply:
The keyword "Test" is prefixed and dumped in the log by test runner... you only need to include the test after than in the test comment. In your case it would look something like:

Test Test that "ceph fs swap" ....

Reply:
That's the not the case in following case  -
```
2023-04-18T16:08:55.406 INFO:tasks.cephfs_test_runner:Starting test: test_mirroring_peer_commands (tasks.cephfs.test_admin.TestMirroringCommands)
test adding and removing peers to a mirror enabled filesystem
2023-04-18T16:08:55.406 INFO:tasks.cephfs_test_runner:test_mirroring_peer_commands (tasks.cephfs.test_admin.TestMirroringCommands)
```

Reply:
Is this with vstart_runner?